Interfaces/API/Dirac.py

```diff
@@ -306,8 +306,8 @@ def submitJob( self, job, mode = 'wms' ):
     if isinstance( job, basestring ):
       if os.path.exists( job ):
         self.log.verbose( 'Found job JDL file %s' % ( job ) )
-        fd = os.open( job, 'r' )
-        jdlAsString = fd.read()
+        f = open( job, 'r' )
```

Comment:
pylint will complain if you name your variable 'f'. Make it at least 2 letters please